geometry_msgs/Pose pose

float64 markerPixelWidth

string IDString

# Debug
float64[] mags
float64[] phases
float64[] phaseVariances

string bitChunksStr
string decodedPayloadStr